frontend/packages/dev-console/src/components/NamespacedPage.tsx

```diff
@@ -13,6 +16,8 @@ export enum NamespacedPageVariants {
 export interface NamespacedPageProps {
   disabled?: boolean;
   hideApplications?: boolean;
+  topologyView?: boolean;
```

Comment:
Make this generic.
Let the `NamespacedPage` accept a prop of `toolbar` where you can pass in react components that get placed in the right side. That way this page has no specific tie to topology.